py/rbv.py

```
@@ -0,0 +1,56 @@
+    if not root:
+        print("No .beads/ found in any parent directory", file=sys.stderr)
+        sys.exit(1)
+
+    jsonl = os.path.join(root, ".beads", "beads.jsonl")
+
+    # Fresh export from Dolt
+    try:
+        os.remove(jsonl)
+    except FileNotFoundError:
+        pass
+    r = subprocess.run(["bd", "export", "--no-memories", "-o", jsonl], cwd=root)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)